draft-ietf-teep-architecture.md

```diff
@@ -1347,6 +1347,33 @@ still a threat present in allowing a potentially compromised TAM to install
 any TA it wants; for example, a known-compromised TA or an explicitly
 malicious TA.
 
+The device manufacturer may want to include a notification support that can allow
+Device Administrators be notified for incidents upon detection with some mitigation 
+methods as suggested above.
+
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs where a TAM turns to act out of 
+expectation of Device Administrators, for example, pushing out TAs that contain 
+some data collection or use users' device resources for distributed jobs for a TAM. 
+To mitigate this threat, TEEs and Device Administrators have several options for 
+detecting and mitigating a hostile TAM:
+
+1. ACLs: Apply an ACL to TAMs, making explicit which TAs the TAM is permitted to install. 
+This limits the scope of a hostile TAM's impact.
+2. Trust Anchor update: when a Trust Anchor entry in the TEE Trust Anchor list is specific to 
+the hostile TAM, a Trust Anchor update process may be used to remove the Trust Anchor as a way 
+to stop the TAM for installing TAs to the TEE.
```

Comment:
```suggestion
to stop the TAM from installing TAs to the TEE.
```
But the doc says "Trust Anchors can be updated similarly to the Personalization Data for any other TA."
so a hostile TAM might modify the trust anchor set to remove any other trust anchors from being able to modify the trust anchor set.